tests/stdout_dummy.py

```diff
@@ -0,0 +1,2 @@
+print("hi")
+print("there")
```

Comment:
I don't think these dummy files are worth it, they currently aren't being used anywhere so I assume they're for subprocesses, but this can be easily doable by passing these as strings directly with the process initialization, separate files just for this seems like a bad idea.

Reply:
I agree, I was going to remove them in the next commit, however didn't get to it

Reply:
Fixed